cli/cmd/datastore_cmd.go

```diff
@@ -13,9 +10,7 @@ var dataStoreCmd = &cobra.Command{
 	Long:   "Manage your tracetest data stores",
 	PreRun: setupCommand(),
 	Run: func(cmd *cobra.Command, args []string) {
-		analytics.Track("Datastore", "cmd", map[string]string{})
```

Comment:
Removed those analytics events because there were useless for us. Besides, we are going to remove analytics from the CLI in a near future.